bitnami/airflow/README.md

```diff
@@ -93,20 +93,23 @@ The following tables lists the configurable parameters of the Airflow chart and
 | `airflow.baseUrl`                         | URL used to access to airflow web ui                                                                 | `nil`                                                        |
 | `airflow.worker.port`                     | Airflow Worker port                                                                                  | `8793`                                                       |
 | `airflow.worker.replicas`                 | Number of Airflow Worker replicas                                                                    | `2`                                                          |
+| `airflow.worker.resources`                | CPU/Memory resource requests/limits                                                                  | `{}`                                                         |
```

Comment:
In my previous comment, I suggested you documenting requests & limits separately. Please follow this approach:
```
| `airflow.worker.resources.limits`    | The resources limits for the Worker containers        | `{}`                        |
| `airflow.worker.resources.requests`  | The requested resources for the Worker containers     | `{}`                        |
```